cdap-ui/app/directives/widget-container/widget-input-schema/widget-input-schema.html

```diff
@@ -52,6 +53,12 @@
           errors="errors[inputschema.name]">
         </my-complex-schema>
       </div>
+      <div
+        ng-if="inputschema.isMacro"
+        ng-class="{'hide': MyInputSchemaCtrl.currentIndex !== $index, '': MyInputSchemaCtrl.currentIndex === $index}"
```

Comment:
Possibly dumb question: Why do we need the second part of this expression in ng-class?

Reply:
Thats not a dumb question at all. Dumb code which needs to be cleaned up. 🤦